app/uploaders/visual_uploader.rb

```diff
@@ -0,0 +1,22 @@
+class VisualUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+  storage :file
+
+  def store_dir
+    "images/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  process resize_to_fill: [360, 360]
+
+  version :thumb do
+    process resize_to_fill: [100, 100]
+  end
+
+  def extension_white_list
+    %w(jpg jpeg gif png)
+  end
+
+  def default_url(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `default_url(*)` if you want the method to accept any arguments but don't care about them.